src/main/java/com/spacesongr/songr/controllers/SongController.java

```diff
@@ -0,0 +1,33 @@
+package com.spacesongr.songr.controllers;
+
+import com.spacesongr.songr.Repository.AlbumRepository;
+import com.spacesongr.songr.Repository.SongRepository;
+import com.spacesongr.songr.models.Album;
+import com.spacesongr.songr.models.Song;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.servlet.view.RedirectView;
+
+@Controller
+public class SongController {
+    @Autowired
+    SongRepository songRepository;
+    @Autowired
+    AlbumRepository albumRepository;
```

Comment:
`albumRepository` is an instance of the class `AlbumRepository`. Because `findByTitle` is not a static method, you need to call the method on your instance, not the class. So for example, line 23 below should read:

`if (albumRepository.findByTitle(AlbumTitle) != null)`

This is also why we generally name instances starting with lower-case letters, so you can quickly see in your code whether you're working with the AlbumRepository class or this albumRepository instance.